.github/workflows/run-unit-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [14.x]
```

Comment:
Currently `cdk synth` (hence unit test) is failing with node16, due to an unknown error arising with esbuild and sharp library. We are still investigating the root cause behind this issue. 
Backlog: **Support build using node16**.